tests/tasks/assert_varlogmessages.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Get content of {{ __default_system_log }}
+  command: cat {{ __default_system_log }}
+  register: __default_system_log_content
+  changed_when: false
+  no_log: true
+
+# /var/log/messages sometimes contains errors not visible in journalctl
```

Comment:
Can you give me an example of this happening?  It seems like a journald configuration issue e.g. journal logs are being trimmed too quickly.  Note that all log messages in /var/log/messages come from journal

Reply:
Currently, `tests_ovirt_elasticsearch.yml` prints error messages only on EL 7. I fixed other tests in this PR and I am looking into this.
```
2024-05-29T11:46:14.984650-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.cacert' file /etc/rsyslog.d/es-ca.crt couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
2024-05-29T11:46:14.984810-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.mycert' file /etc/rsyslog.d/es-cert.pem couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
2024-05-29T11:46:14.984979-04:00 ibm-p8-kvm-03-guest-02 systemd: Started System Logging Service.
2024-05-29T11:46:14.985136-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.myprivkey' file /etc/rsyslog.d/es-key.pem couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
2024-05-29T11:46:14.985322-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.cacert' file /etc/rsyslog.d/es-ca.crt couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
2024-05-29T11:46:14.985492-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.mycert' file /etc/rsyslog.d/es-cert.pem couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
2024-05-29T11:46:14.985633-04:00 ibm-p8-kvm-03-guest-02 rsyslogd: error: 'tls.myprivkey' file /etc/rsyslog.d/es-key.pem couldn't be accessed: No such file or directory  [v8.24.0-57.el7_9.3 try http://www.rsyslog.com/e/2039 ]
```